Doc fixes (#1390)
dfinity/portal: docs/developer-docs/setup/install/index.mdx

## What gets installed

The IC SDK installation script installs several components in default locations on your local computer. The following table describes the development environment components that the installation script installs:

| Component    | Description                                                                                        | Default location                              |
|--------------|----------------------------------------------------------------------------------------------------|-----------------------------------------------|
| dfx          | Command-line interface (CLI)                                                     | `/usr/local/bin/dfx`                          |

> | cdk-rs       | Rust CDK                                                       | `/usr/local/bin/dfx`                          |

Reviewer: cdk-rs isn't actually installed by, or contained within, dfx; this one should probably be reverted

Author: removed

| replica      | Internet Computer local network binary                                                             | `~/.cache/dfinity/versions/<VERSION>/replica` |
| uninstall.sh | Script to remove the SDK and all of its components                                    | `~/.cache/dfinity/uninstall.sh`               |
| versions     | Cache directory that contains a subdirectory for each version of the SDK you install. | `~/.cache/dfinity/versions`                   |

## SDK vs CDK vs `dfx`

There are a few components worth expanding on:

> 1. **IC SDK** - This is the SDK created by DFINITY for creating and managing canisters. This is just one SDK in ICP community and a common entry point for developers.

Reviewer 2: ```suggestion
1. **IC SDK** - This is the SDK created by DFINITY for creating and managing canisters. This is just one SDK in the ICP community and a common entry point for developers.
```

> 2. **Canister Development Kit (CDK)** - A CDK is an adapter used by the IC SDK so a programming language has the features needed to create and manage canisters. 

Reviewer 2: ```suggestion
2. **Canister Development Kit (CDK)** - A CDK is an adapter used by the IC SDK that provides a programming language with the features necessary to create and manage canisters. 
```

2. **Canister Development Kit (CDK)** - A CDK is an adapter used by the IC SDK so a programming language has the features needed to create and manage canisters. 

> The IC SDK comes with a few CDKs already installed for you so you can use them in the language of yoru choice. That is why there is a [Rust CDK](https://github.com/dfinity/cdk-rs), [Python CDK](https://demergent-labs.github.io/kybra/), 

Reviewer: ```suggestion
The IC SDK comes with a few CDKs already installed for you so you can use them in the language of your choice. That is why there is a [Rust CDK](https://github.com/dfinity/cdk-rs), [Python CDK](https://demergent-labs.github.io/kybra/), 
```
This wording seems to either imply that Kybra support comes built-in, or that Rust doesn't.

Author: Ok i rewrote it. lmk what you think @adamspofford-dfinity ?

> [TypeScript CDK](https://demergent-labs.github.io/azle/), etc... Since CDKs are components used the SDK, some developer choose to use the CDK directly (without the IC SDK), 

Reviewer: ```suggestion
[TypeScript CDK](https://demergent-labs.github.io/azle/), etc... Since CDKs are components used by the SDK, some developers choose to use the CDK directly (without the IC SDK), 
```
